bin/start_be.sh

```diff
@@ -16,11 +16,38 @@
 # specific language governing permissions and limitations
 # under the License.
 
-curdir=$(dirname "$0")
-curdir=$(
-    cd "$curdir"
-    pwd
-)
+#check max_map_count
+MAX_MAP_COUNT=`sysctl -n vm.max_map_count`
+if [ $MAX_MAP_COUNT -lt 2000000 ]; then
```

Comment:
Use `[[]]` to test, otherwise, you should double quote $MAX_MAP_COUNT.

```suggestion
if [[ "${MAX_MAP_COUNT}" -lt 2000000 ]]
```